base_layer/core/src/transactions/transaction.rs

```diff
@@ -295,12 +295,12 @@ impl UnblindedOutput {
             sender_offset_public_key: self.sender_offset_public_key.clone(),
             metadata_signature: self.metadata_signature.clone(),
         };
-        // A range proof can be constructed for an invalid value so we should confirm that the proof can be verified.
-        if !output.verify_range_proof(&factories.range_proof)? {
-            return Err(TransactionError::ValidationError(
-                "Range proof could not be verified".into(),
-            ));
-        }
+        // // A range proof can be constructed for an invalid value so we should confirm that the proof can be verified.
```

Comment:
> `// range proof can be constructed for an invalid value so we should confirm that the proof can be verified.`

Assuming the invalid value talked about here is a negative value? Something that is already an invariant because Microtari is backed by a u64.